manual/graphics/tutorials/index.md

```diff
@@ -4,7 +4,7 @@ This section contains a set of tutorials related to graphics.
 
 * [How to render object outline](render-object-outline.md)
 * [How to render FPS weapon](render-fps-weapon.md)
-* [How to se anisotropic texture sampler](anisotropic-texture-sampler.md)
+* [How to Use anisotropic texture sampler](anisotropic-texture-sampler.md)
```

Comment:
Can you change it to lowercase `use` to match others?

Reply:
oh my bad fixed it :D